packages/entities/entities-certificates/vite.config.ts

```diff
@@ -16,6 +16,18 @@ const config = mergeConfig(sharedViteConfig, defineConfig({
       entry: resolve(__dirname, './src/index.ts'),
       fileName: (format) => `${sanitizedPackageName}.${format}.js`,
     },
+    rollupOptions: {
+      external: [
+        '@kong/icons',
+        '@kong-ui-public/entities-shared',
+      ],
```

Comment:
Note that adding `@kong-ui-public/entities-shared/dist/style.css` here does not impact bundle size because they are already externalized based on the package name.

cc @2eha0

Reply:
These packages should/could actually be externalized in the root vite.config

Reply:
> Note that adding `@kong-ui-public/entities-shared/dist/style.css` here does not impact bundle size because they are already externalized based on the package name.
> 
> cc @2eha0

As you said, when I added `@kong-ui-public/entities-shared/dist/style.css` just based on your commit and rebuilt, It does reduce the size of the `dist/style.css`
```
dist/style.css                      1.37 kB │ gzip:  0.39 kB
dist/entities-certificates.es.js  336.65 kB │ gzip: 69.75 kB
dist/style.css                       1.37 kB │ gzip:  0.39 kB
dist/entities-certificates.umd.js  246.68 kB │ gzip: 60.12 kB
```
This is clearly stated in the [rollup documentation](https://rollupjs.org/configuration-options/#external)
> the name of an external dependency, exactly the way it is written in the import statement. I.e. to mark import "dependency.js" as external, use "dependency.js" while to mark import "dependency" as external, use "dependency".

Since we wrote `import '@kong-ui-public/entities-shared/dist/style.css'` in the code so we need use `@kong-ui-public/entities-shared/dist/style.css`

It is not hard to have a try.

Reply:
Yes I misspoke in the original comment. We should likely use a regular expression here to capture any imports from the package instead of having to list each individually.